### PR TITLE
Update the list filter of WorkflowLevel2 view

### PR DESCRIPTION
**Pull request by jefmoura**
## Purpose
OrgAdmins should be able to view all WFL2 items associated with the Org's WFL1

Related ticket: TolaDataV2/[#511](https://github.com/toladata/TolaDataV2/issues/511)

### CODE REVIEW

File: feed/tests/test_workflowlevel2view.py [hunk 1]
```diff
@@ -16,38 +16,39 @@ def setUp(self):
         self.factory = APIRequestFactory()
         self.tola_user = factories.TolaUser()
 
-    def test_list_workflowLlevel2_superuser(self):
-        request = self.factory.get('/api/workflowLlevel2/')
+    def test_list_workflowLevel2_superuser(self):
```
Comment (Menda) [inline feed/tests/test_workflowlevel2view.py]: I know it's stupid, but in python for variable (except classes) or functions naming we don't use uppercase

File: feed/tests/test_workflowlevel2view.py [hunk 2]
```diff
@@ -16,38 +16,39 @@ def setUp(self):
...
+        request = self.factory.get('/api/workflowLevel2/')
```
Comment (Menda) [inline feed/tests/test_workflowlevel2view.py]: The original URL is with lowercase

File: feed/tests/test_workflowlevel2view.py (hunk 1 above)
Comment (jefmoura) [reply feed/tests/test_workflowlevel2view.py]: I know, I'm gonna change it.

File: feed/tests/test_workflowlevel2view.py (hunk 2 above)
Comment (jefmoura) [reply feed/tests/test_workflowlevel2view.py]: You're right. Even though the function doesn't use the URL.

**Pull request merged**